### PR TITLE
scripts: don't build platform during build metasystem; simplify platform install; rename scripts corresponding to components names

### PR DESCRIPTION
**Pull request by NikitaZotov**
Dependencies:

- https://github.com/ostis-ai/sc-machine/pull/372
- https://github.com/ostis-ai/sc-component-manager/pull/88
- https://github.com/ostis-ai/ostis-web-platform/pull/108

### CODE REVIEW

File: scripts/build_ps.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  source "${CURRENT_DIR}/set_vars.sh"
+fi
+
+stage "Build cxx problem solver"
+
+BUILD_SCRIPTS="${SC_MACHINE_PATH}/scripts/build-scripts"
+"${BUILD_SCRIPTS}/build_cxx_project.sh" -p "${METASYSTEM_CMAKE_PATH}" -s "${METASYSTEM_CXX_SOURCES_PATH}" \
+  -b "${METASYSTEM_BUILD_PATH}" --cmake-arg "-DSC_BIN_PATH=${BINARY_PATH}" "$@"
+
+stage "CXX problem solver is built"
```
Comment (MksmOrlov) [inline scripts/build_ps.sh:19]:
```suggestion
stage "cxx problem solver is built"
```

File: scripts/build_ps.sh
Comment (MksmOrlov) [inline scripts/build_ps.sh:1]: I'd like to separate build, install and run scripts to directories in scripts/

**Review by MksmOrlov — CHANGES_REQUESTED** [review]

**Review by MksmOrlov — CHANGES_REQUESTED** [review]
image

File: scripts/build_ps.sh
Comment (NikitaZotov) [reply scripts/build_ps.sh:1]: I have idea to separate only core scripts, but not user scripts. It can be inconvient for users

**Comment by NikitaZotov** [comment]
> image

You test some another branch

**Review by MksmOrlov — APPROVED** [review]

**Pull request merged**